content/6-design-tools.md

```diff
@@ -139,9 +139,9 @@ Accessibility isn't just a feature - it's a fundamental aspect of good design th
 
 Helpful accessibility tools include:
 
-- [WAVE](https://wave.webaim.org/){:target="_blank"}: Evaluate web content for accessibility issues
-- [Contrast Checker](https://webaim.org/resources/contrastchecker/){:target="_blank"}: Ensure your color choices meet accessibility standards
-- [Axe](https://www.deque.com/axe/){:target="_blank"}: Automated accessibility testing
+- [Accessibility Insights](https://accessibilityinsights.io/){:target="_blank"}: Evaluate web content in your browser for accessibility issues (using Deque's axe-core)
+- [Oobee](https://github.com/GovTechSG/oobee){:target="_blank"}: Site-wide accessibility testing (using Deque's axe-core)
```

Comment:
If I see it correctly, this requires development experience to set up, which makes the tool a lot harder to use. Maybe we keep the link to the axe browser extension?